src/components/chat/comment-textbox.tsx

```diff
@@ -147,7 +147,7 @@ export const CommentTextBox: React.FC<IProps> = (props) => {
             Object.keys(commentTags).map(key => {
               const value = commentTags[key];
               return (
-                <option key={key} value={key}> {value} </option>
+                <option key={key} value={value}> {value} </option>
```

Comment:
We actually want to write the `value` into firestore, otherwise what gets rendered in the white comment tag box shows for example "part-to-part" in lower case and not "Part-to-Part"

Reply:
I don't think this is correct. We do want to write `key` to firestore. Then when you render the comment tag box, you'll want to use what's in firestore as an index, rather than writing it directly.

Reply:
But if we write the actual value into firestore and then the author changes the value, the old tags stored in firestore won't work anymore.